srcpkgs/didder/template

```diff
@@ -0,0 +1,22 @@
+# Template file for 'didder'
+pkgname=didder
+version=1.1.0
+revision=1
+build_style=go
+go_import_path="github.com/makeworld-the-better-one/didder"
+go_ldflags="-s -w -X main.version=${version} -X main.builtBy=Void"
+hostmakedepends="pandoc"
+short_desc="Command-line image dithering tool"
+maintainer="Dakota Walsh <kota@nilsu.org>"
+license="GPL-3.0-or-later"
+homepage="https://github.com/makeworld-the-better-one/didder"
+changelog="https://github.com/makeworld-the-better-one/didder/blob/main/CHANGELOG.md"
+distfiles="https://github.com/makeworld-the-better-one/${pkgname}/archive/refs/tags/v${version}.tar.gz"
+checksum=a850ae54125ed2aa1be54b1cc11481c00094d346861bd59014d621dc48f9aed6
+
+post_install() {
+	pandoc didder.1.md -s -t man -o didder.1
+	sed -i 's/VERSION/$(VERSION)/g' didder.1
+	sed -i 's/DATE/$(DATE)/g' didder.1
```

Comment:
```roff
.TH "DIDDER" "1" "$(DATE)" "didder $(VERSION)" "User Manual"
```
this isn't doing what you think it is. those are `make` variable substitutions.

```suggestion
	sed -i "s/VERSION/${version}/g" didder.1
	_date="$(date --utc --date "@$SOURCE_DATE_EPOCH" "+%B %d, %Y")"
	sed -i "s/DATE/${_date}/g" didder.1
```
this should insert the actual date and version

Reply:
Thanks, didn't realize that wasn't even working!